src/server/controllers/logout.controller.js

```diff
@@ -0,0 +1,8 @@
+const serverConf = require('../conf/server');
+
+module.exports = (app) => {
+    app.get('/logout', (req, res) => {
+        res.clearCookie(serverConf.sessionCookieName);
```

Comment:
Makes sense that it should be in auth. I replaced `res.clearCookie` with `req.session.destroy`.  `req.logout` only removes the passport object attached to session. The session/cookie is still passed around after logout, which was cleared by `clearCookie`. I just learned that `req.session.destroy` is better practice than `clearCookie`.